Makefile

```diff
@@ -23,9 +23,9 @@ fix-go:  # auto-fixes all Go lint issues
 	gofmt -s -w ./src ./test
 
 fix-md:  # auto-fixes all Markdown lint issues
-	${CURDIR}/tools/prettier/node_modules/.bin/prettier --write .
+	dprint fmt
```

Comment:
I'm switching all my codebases to https://dprint.dev. It's a rewrite of Prettier in Rust. Two orders of magnitude faster.